### ISSUE TITLE
Chromecast possiblity

### ISSUE THREAD

**Issue by gomme600** [Dusk-Labs/dim#606]
**Is your feature request related to a problem? Please describe.**
The ability to cast media to a Chromecast

**Describe the solution you'd like**
A cast button on the website.

**Describe alternatives you've considered**
Unfortunately there is no alternative.

**Additional context**
Would it (or is it already) possible to cast media to a Chromecast? I can't find the option anywhere on the webapp and it doesn't seem to be mentioned in the documentation.

I personally cast all my media, I don't watch movies on my phone so this seems like a big oversight.

Thanks for any insight.

**Comment by inspire22** [comment]
Yeah I saw this project too but also need a way to get it to my TV.  Last release was in 2001 though so I worry it's not still active.  Feels like all the alternatives suck though, was hoping this would be the one.
